### ISSUE TITLE
Sorting descending by default

### ISSUE THREAD

**Issue by mdmota** [sonnemaf/ReflectionIT.Mvc.Paging#22]
When loading the page, I need my list to be sorted in descending order by a certain field.
When adding the sortExpression and defaultSorteExpression the list always enters sorted ascending.

Any solution?

**Comment by sonnemaf** [comment]
There was a bug which I just fixed and is now included in version 3.3.1.0 (can be found on NuGet). You can now add a minus sign to the sortColumn parameter of the SortableHeaderFor() extension method. See example below.

The ProductName and CompanyName columns are ordered descending because it starts with a "-".

```xml
<table class="table">
    <tr>
        <th>
            @Html.SortableHeaderFor(model => model.ProductName, "-ProductName")
        </th>
        <th>
            @Html.SortableHeaderFor(model => model.Supplier.CompanyName, "-Supplier.CompanyName, ProductName")
        </th>
        <th>
            @Html.SortableHeaderFor(model => model.Category.CategoryName, "Category.CategoryName, ProductName")
        </th>
        <th>
            @Html.SortableHeaderFor(model => model.UnitPrice)
        </th>
        <th></th>
    </tr>

    @foreach (var item in Model) {
```

Please let me know if it works.

**Comment by mdmota** [comment]
Thank you soo much!
